client/src/partials/patients/registry/registry.js

```diff
@@ -14,60 +14,68 @@ PatientRegistryController.$inject = [
  */
 function PatientRegistryController($translate, Patients, $uibModal) {
   var vm = this;
-  // options for the UI grid
   
   vm.search = search;
   vm.momentAge = momentAge;
-    
+ 
+  // track if module is making a HTTP request for patients
+  vm.loading = false;
+  
   /** TODO MANAGE COLUMN : LAST_TRANSACTION */
+  // the column attribute `displayName` must be used in favour of `name` in order to allow `headerCellFilter` to function
   vm.uiGridOptions = {
-    appScopeProvider : vm, // ensure that the controller's `this` variable is bound to appScope
+    appScopeProvider : vm,
     enableColumnMenus : false,
+    headerCellFilter: 'translate',
     columnDefs : [
-      { field : 'reference', name : $translate.instant('TABLE.COLUMNS.REFERENCE') },
-      { field : 'patientName', name : $translate.instant('TABLE.COLUMNS.NAME') },
-      { field : 'patientAge', name : $translate.instant('TABLE.COLUMNS.AGE') },
-      { field : 'sex', name : $translate.instant('TABLE.COLUMNS.GENDER') },
-      { field : 'hospital_no', name : $translate.instant('TABLE.COLUMNS.HOSPITAL_FILE_NR') },
-      { field : 'registration_date', cellFilter:'date', name : $translate.instant('TABLE.COLUMNS.DATE_REGISTERED') },
-      { field : 'last_visit', cellFilter:'date', name : $translate.instant('TABLE.COLUMNS.LAST_VISIT') },
-      { field : 'dob', cellFilter:'date', name : $translate.instant('TABLE.COLUMNS.DOB') }  
+      { field : 'reference', displayName : 'TABLE.COLUMNS.REFERENCE', headerCellFilter: 'translate' },
```

Comment:
How does the performance of the headerCellFilter compare to the `$translate.instant()`?